src/web-components/nav-menu/nav-menu.html

```diff
@@ -1,6 +1,12 @@
-<button><img src="../../assets/icon-menu.svg" alt="Open Menu" /></button>
+<button class="open-nav-btn">
```

Comment:
als je een button component hebt gemaakt, zou je die bijv. hier ook kunne gebruiken in je html